File: 2022/01/js/1.js
const fs = require("fs");
let data = fs.readFileSync("i.txt").toString();
let p = data.split(/\r?\n\r?\n/)
let r = p.map(r => r.split(/\r?\n/).reduce((acc, n) => acc + parseInt(n, 10), 0)).reduce((acc, n) => Math.max(acc, n), 0);
console.log(r)

File: 2022/01/py/1.py
with open("i.txt") as f:
    data = [l.removesuffix("\n") for l in f]

m = 0
s = 0
for l in data:
    if l == "":
        s = 0
    else:
        s += int(l)
        m = max(s, m)

print(m)